app/views/cookers/show.html.erb

```diff
@@ -18,10 +18,20 @@
 
   <h1>RENDER MODAL ZITH FORM HERE</h1>
 
-  <%= link_to new_user_booking_path(@cooker,@booking) do %>
-    <div class="text-center btn-effect">
```

Comment:
The booking-new form (when you click on “book me!” should come up as a bootstrap modal, not be shown directly in the show page. We'll have to keep the "book me!" button here (you can take a look at the Figma!).